code/modules/psionics/psionic_items/psi_Harmor.dm

```diff
@@ -105,7 +104,6 @@
 	icon_state = "gloves"
 	icon_override = 'icons/obj/psionic/occHmob.dmi'
 	slot_flags = SLOT_GLOVES
-	item_flags = THICKMATERIAL
```

Comment:
Are you sure?

Reply:
all psi armor has THICKMATERIAL, including the light/void armor, so heavy should too.

Reply:
intended? Yes and no. This one is correct...its slot flags where defined 2 times, Harmless but still bad clutter, However I removed both by mistake on some

Reply:
item flags*

Reply:
ah this is the gloves. Gloves simply dont need thickmaterial defined. It serves no function as far as I'm aware thickmaterial only does anything for chest/helmets